matRad_calcPhotonDose.m

```diff
@@ -91,7 +91,7 @@
 [yCoordsV_vox, xCoordsV_vox, zCoordsV_vox] = ind2sub(ct.cubeDim,V);
 
 % set lateral cutoff value
-lateralCutoff = 65; % [mm]
+lateralCutoff = 82; % [mm]
```

Comment:
the value 82mm is taken from the paper and refers to a error of <4% of integral dose of large fields. This value is, similar to the cut off used for the particle dose calculation, a trade off between performance and accuracy. The question is what is a good default value for matRad. Maybe its better to change it back 65mm.